kitchen.dokken.yml

```diff
@@ -42,6 +43,12 @@ platforms:
       pid_one_command: /sbin/init
       volumes:
         - /sys/fs/cgroup:/sys/fs/cgroup:ro  # required by systemd
+    provisioner:
+      attributes:
+        splunk:
+          server:
+            url: https://download.splunk.com/products/splunk/releases/8.0.9/linux/splunk-8.0.9-153839c8b72f-linux-2.6-x86_64.rpm
+            version: 8.0.9
```

Comment:
Why is this needed?

Reply:
This change is noted in the commit message: commit b0b8b5b3b293e5860afea481b870e3a100300a63

```
Author: Dang H. Nguyen <dang.nguyen@disney.com>
Date:   Fri May 14 16:13:02 2021 -0700

    for the shdeployer test, failure was due to CentOS7 RPM corruption from Splunk; therefore, this change will use the 8.0.9 RPM instead
```

Changing this attribute value in the cookbook would be a breaking change, per semver, so leaving the default attribute value the way it is and adding this in the Test Kitchen environment is a decent workaround.